packages/modules/display_themes/colors/source/src/components/chargePointList/configPanels/ConfigEco.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<div class="ecosettings">
+		<PriceChart
+			v-if="etData.active && etData.etPriceList.size > 0"
+			:charge-point-id="props.chargepoint.id"
+			class="fullwidth"
+		/>
+		<span class="fullwidth mb-4">
+			<button type="button" class="btn btn-secondary" @click="pageDown">
+				<i class="fa-solid fa-circle-down me-2"></i>
+				Abwärts
+			</button>
+		</span>
+
+		<!-- Minimal ampere -->
+		<ConfigItem
+			title="Minimaler Ladestrom unter der Preisgrenze"
+			icon="fa-bolt"
+			:fullwidth="true"
+		>
+			<RangeInput
+				id="minCurrent"
+				v-model="cp.ecoMinCurrent"
+				:min="6"
+				:max="32"
+				:step="1"
+				unit="A"
+			/>
+		</ConfigItem>
+
+		<!-- Phases -->
+		<ConfigItem title="Anzahl Phasen" icon="fa-plug" :fullwidth="true">
+			<RadioInput
+				id="targetPhases"
+				v-model="cp.ecoTargetPhases"
+				:options="[
+					['Eine', 1],
+					['Alle', 2],
```

Comment:
Bitte bei den Bezeichnungen an den Einstellungen orientieren. Dort wird "Maximum" verwendet (kann auch "Max." abgekürzt werden, falls es eng wird). "Alle" könnte dazu verleiten, immer an drei Phasen zu denken, was aber nicht stimmt.

Analog bitte auch in den anderen "ConfigXXX.vue".